src/zenml/zen_stores/models/user_management_models.py

```diff
@@ -107,20 +107,33 @@ class Team(BaseModel):
     name: str
 
 
+def get_active_user_id() -> UUID:
+    from zenml.repository import Repository
+
+    user = Repository().active_user
+    if Repository().zen_store.get_user(user.name).id != user.id:
```

Comment:
Is this check really necessary? The `Repository().active_user` is essentially also a call to `Repository().zen_store.get_user`, so this will never fail right?

Reply:
Well spotted, I guess that got left over from the rebase.